core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -225,13 +233,50 @@ oppia.directive('audioTranslationBar', [
             $scope.showRecorderWarning = false;
           };
 
+          $scope.stopRecording = function() {
+            $scope.voiceoverRecorder.stopRecord();
+            $scope.recordingComplete = true;
+            cancelTimer();
+            $scope.voiceoverRecorder.getMp3Data().then(function(audio) {
+              var fileType = 'audio/mp3';
+              $scope.audioBlob = new Blob(audio, {type: fileType});
+              // free the browser from workers
```

Comment:
Capitalize the first letter in the comment and end with a period(.) like a full sentence. (everywhere)

Reply:
Okay done.